fcrepo-jms/pom.xml

```diff
@@ -47,6 +47,10 @@
       <artifactId>javax.inject</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.modeshape</groupId>
```

Comment:
Also, I suspect that finding where the JCR_CONTENT value is escaping from fcrepo-kernel-modeshape will also help remove that from fcrepo-module-auth-webac: http://git.io/v854c